Gtk-3.0.gir

```diff
@@ -17136,7 +17136,7 @@ or -1 if the attribute is not mapped.</doc>
             <doc xml:space="preserve" filename="gtk/gtkcellarea.c" line="2808">the name of the property to get</doc>
             <type name="utf8" c:type="const gchar*"/>
           </parameter>
-          <parameter name="value" transfer-ownership="none">
+          <parameter name="value" transfer-ownership="none" direction="out" caller-allocates="1">
```

Comment:
That's strange. Shouldn't it be `full` instead of `none`? The caller allocates it, so the caller also has to free it again.

Maybe the `transfer-ownership` annotation is just redundant in this case!

Looks good to me in any case, the resulting code seems fine but I'll shortly double-check

Reply:
I.e. get this in :)

Reply:
So I merge or not? :)

Reply:
It's consistent with other functions and the generated code is correct